rustdesk-server/umbrel-app.yml

```
@@ -0,0 +1,33 @@
+manifestVersion: 1
```

Update manifest version to support hooks.
```suggestion
manifestVersion: 1.1
```